### PR TITLE
Fix "create CustomType" form

### PR DESCRIPTION
**Pull request by hypervillain**
https://linear.app/prismic/issue/SM-200/aauser-in-sm-i-cant-save-a-ct-without-name-or-ids

- Check for empty ID values
- Set label as ID of not set
- Fix a regression that inverted the condition of "create CT" display

### CODE REVIEW

File: packages/slice-machine/tests/project/cypress/integration/pages/cts.spec.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+describe("onboarding skip", () => {
+  it('displays create custom type button and modal', () => {
+    cy.visit('/')
+  
+    cy.get('[create-ct]').click()
```
Comment (MarcMcIntosh) [inline packages/slice-machine/tests/project/cypress/integration/pages/cts.spec.js]: missing `data-cy` part of the selector

**Review by MarcMcIntosh — CHANGES_REQUESTED** [review]

File: packages/slice-machine/tests/project/cypress/integration/pages/cts.spec.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+describe("onboarding skip", () => {
```
Comment (MarcMcIntosh) [inline packages/slice-machine/tests/project/cypress/integration/pages/cts.spec.js]: bad name as well

**Review by MarcMcIntosh — CHANGES_REQUESTED** [review]

File: packages/slice-machine/tests/project/cypress/integration/pages/cts.spec.js (hunk 1 above)
Comment (Miniplop) [reply packages/slice-machine/tests/project/cypress/integration/pages/cts.spec.js]: Fixed

File: packages/slice-machine/tests/project/cypress/integration/pages/cts.spec.js (hunk 2 above)
Comment (Miniplop) [reply packages/slice-machine/tests/project/cypress/integration/pages/cts.spec.js]: Fixed

**Review by MarcMcIntosh — APPROVED** [review]

**Pull request merged**